src/usb-export

```diff
@@ -1,4 +1,5 @@
-#!/bin/sh
+#!/bin/bash --
+set -euo pipefail
```

Comment:
While generally a good idea, have you reviewed the whole script for any things that could fail (and for each - decide if failure should really be fatal)? And same for uninitialized variables.

I'll give it a go on openQA, but manual review would be useful too.

Reply:
In most cases, failure means that something is seriously wrong and we are in an undefined state.  I would prefer to leave uninitialized variable checks to shellcheck.

Reply:
I checked all of them and I think the remaining ones should abort the script.  The cases that I believe should not have been fixed.